cumulusci/robotframework/CumulusCI.py

```diff
@@ -114,6 +116,45 @@ def login_url(self, org=None):
             org = self.keychain.get_org(org)
         return org.start_url
 
+    def _refresh_community_info_cache(self):
+        response = self.sf.restful("connect/communities")
+        self._community_info_cache = {
+            community["name"]: community for community in response["communities"]
```

Comment:
I didn't know this for sure, but I validated in an org that the Name field is required to be unique.